README.md

```diff
@@ -1,7 +1,6 @@
 # laminas-cache-storage-adapter-test
 
-[![Build Status](https://travis-ci.com/laminas/laminas-cache-storage-adapter-test.svg?branch=master)](https://travis-ci.com/laminas/laminas-cache-storage-adapter-test)
-[![Coverage Status](https://coveralls.io/repos/github/laminas/laminas-cache-storage-adapter-test/badge.svg?branch=master)](https://coveralls.io/github/laminas/laminas-cache-storage-adapter-test?branch=master)
+[![Build Status](https://github.com/laminas/laminas/laminas-cache-storage-adapter-test/workflows/Continuous%20Integration/badge.svg)](https://github.com/laminas/laminas/laminas-cache-storage-adapter-test/actions?query=workflow%3A"Continuous+Integration")
```

Comment:
https://github.com/laminas/laminas-cache/blob/64e2f156b2d97180f05f1bcf1be359e9297da8e8/README.md

The link to the image as well as to the result has to be changed according to the documentation:
https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-workflow-file-name

Reply:
I'm sorry I cannot spot any error here, where is it?

Reply:
Both notation are working, `continous-integration.yml` and `Continuous%20Integration`. For example: https://github.com/laminas/laminas-composer-autoloading#readme

But only one version is documented: `https://github.com/<OWNER>/<REPOSITORY>/actions/workflows/<WORKFLOW_FILE>/badge.svg`

Reply:
Ah ok, I just followed @weierophinney guide: https://gist.github.com/weierophinney/9decd19f76b7d9745c6559074053fa65
I'll update this PR in few minutes then

Reply:
Don't worry, the badge is missing in [the Gist with the steps to integrate the workflow](https://gist.github.com/weierophinney/b003e50c3c2667d08076caf31ebd36a4). An update is needed there.